HTTPServer.driver/Driver.cs

```diff
@@ -9,29 +9,43 @@ namespace HTTPServer.driver
     public class Driver
     {
         private static Server server = new Server();
+        private static int port = 0;
+        private static string directoryPath = "";
 
         public static void Main(string[] args)
         {
-            server.Start(HandlePort(args), new NetworkSocket());
+            HandleCommands(args);
+            server.Start(port, new NetworkSocket(), directoryPath);
             server.HandleClients();
 
         }
 
-        private static int HandlePort(string[] args)
+        private static void HandleCommands(string[] args)
```

Comment:
There must be a better/cleaner way to parse these arguments than a nested if/else.